app/models/merchant.rb

```diff
@@ -77,5 +77,14 @@ def self.top_five_merchants_by_revenue
     .select('merchants.*, SUM(invoice_items.unit_price * invoice_items.quantity) AS total_revenue')  # Calculate total revenue
     .group('merchants.id') # group records by merchant
     .order('total_revenue DESC')
+    .limit(5) #added because the server was returning more than 5
+  end
+
+  def best_day
+    invoices.joins(:items)
+      .group("invoices.created_at")
+      .sum("invoice_items.quantity * items.unit_price")
+      .max_by { |date, sales| sales }
```

Comment:
this works but I'm not sure if this is best practice to use a ruby method in an ActiveRecord call. It's fine to merge just an area for potential to refactor later on.